### DISCUSSION TITLE
Error generating datamodel ("could not parse datamodel") !help

### DISCUSSION THREAD

**Discussion by jonrusso** [prisma/prisma#17456] [Q&A]
# Howdy folks!

I'm having a problem generating my diagram through two dependencies if I am not wrong which are "prisma-erd-generator @mermaid-js/mermaid-cli" and I have already checked the prisma version along with the client version, as well as the directories in the "node_modules/@prisma/client" folder and yet after entering the command "npx prisma generate" it gives the following error: 
![image](https://user-images.githubusercontent.com/109310127/213820983-5fdac2e6-97dd-41e7-abae-2ff7cce131ae.png)
------
## On this way, I'm not be able to get a file in .svg to read and see this graphic from my database: ##
![image](https://user-images.githubusercontent.com/109310127/213821066-5c8aaa2f-0006-487d-8951-136f187fea68.png)
-----
This is my schema:
 ```ts
generator client {
  provider = "prisma-client-js"
}

generator erd {
  provider = "prisma-erd-generator"
  theme    = "forest"
}

datasource db {
  provider = "sqlite"
  url      = env("DATABASE_URL")
}

model Habit {
  id         String          @id @default(uuid())
  title      String
  created_at DateTime
  dayHabits  DayHabit[]
  weekDays   HabitWeekDays[]

  @@map("habits") 
}

model HabitWeekDays {
  id       String @id @default(uuid())
  habit_id String
  week_day Int
  habit    Habit  @relation(fields: [habit_id], references: [id])

  @@unique([habit_id, week_day])
  @@map("habit_week_days")
}

model Day {
  id        String     @id @default(uuid())
  date      DateTime   @unique
  dayHabits DayHabit[]

  @@map("days")
}

model DayHabit {
  id       String @id @default(uuid())
  day_id   String
  habit_id String
  habit    Habit  @relation(fields: [habit_id], references: [id])
  day      Day    @relation(fields: [day_id], references: [id])

  @@unique([day_id, habit_id])
  @@map("day_habits")
}
```

**Comment by janpio** [comment]
This is an issue or question for `prisma-erd-generator`, not Prisma itself.

**Reply by jonrusso** [reply]
oh! Sure. Maybe on this currently version of prisma (4.9.0) the 'erd-generator' doesn't work as usual in previous patches.

**Reply by janpio** [reply]
Possible, but the author is quite receptive so I bet they can figure this out.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
